rpc/src/module/chain.rs

```diff
@@ -1191,6 +1191,50 @@ pub trait ChainRpc {
     /// ```
     #[rpc(name = "get_block_median_time")]
     fn get_block_median_time(&self, block_hash: H256) -> Result<Option<Timestamp>>;
+
+    /// Returns the median fee rate in recent number of blocks specified by user.
+    ///
+    /// ## Params
+    ///
+    /// * `max_blocks_scan` - the latest x (default: 42) blocks scanned to calculate median fee rate.
+    /// * `min_transaction` - minimal number transactions(default: 42) included in the block.
+    ///
+    /// ## Returns
+    /// * `Result<Capacity>` - the median fee rate(in Shannons).
+    ///
+    /// When the `blocks_to_scan` is greater than `max_blocks_to_scan(default: 42)`, this RPC returns error;
+    /// otherwise returns the median fee rate (in Shannons).
```

Comment:
```suggestion
    /// When there are less than `min_transaction` transactions in the last `max_blocks_scan` blocks, this RPC returns an error. Otherwise, it returns the median fee rate (in Shannons per vbyte).
```